docs/antora-playbook.yml

```diff
@@ -18,7 +18,8 @@ content:
   - url: https://github.com/smallrye/smallrye-fault-tolerance.git
     branches: ~
     tags:
-    - '5.?.?'
+    - '5*'
```

Comment:
Do I get this right that this will create a separate documentation book for every minor AND micro release? Which would potentially be a lot of books?
Isn't it better to use branches per minor release, for example - build from `5.0.x` branch which you can update over time, and that branch will translate into a book with "documentation for 5.0"? And then, when needed, add a 5.1.x branch, etc?

Reply:
That is correct. Anything wrong about that? I don't want to maintain branches as much as possible :-)

Reply:
I don't quite like the idea of having a separate documentation for each micro (bugfix) release. There would potentially be too many of them. Another problem is that if you find an issue in t

Reply:
For Metrics, I have a documentation set for each minor release branch, because I assume that the changes between micro releases are minimal and don't deserve a separate set. To update docs for a particular minor release, I just add stuff into the right branch and regenerate the docs. No tagging or releases needed.

Reply:
I'd be happy to have a single doc set for a minor version, if Antora knew how to select the latest tag for that minor. I know I lose the ability to fix documentation freely, but I also really don't want to maintain branches if it absolutely isn't necessary. (No, Antora limitations doesn't make that necessary :-) )